app/components/DataTable/components/ColumnPicker/__tests__/ColumnPicker.test.tsx

```diff
@@ -0,0 +1,85 @@
+import React from 'react';
+import { screen, render, fireEvent } from '@testing-library/react';
+
+import ColumnPicker from '..';
+
+const mockColumns = [
+  {
+    colId: 'username',
+    isVisible: () => true,
+    name: 'username',
+    visible: true,
+  },
+  {
+    colId: 'password',
+    isVisible: () => false,
+    name: 'password',
+    visible: false,
+  },
+];
+
+const mockLabel = 'Registered Users Table';
+
+describe('ColumnPicker', () => {
+  test('It matches the snapshot', () => {
+    const { asFragment } = render(
+      <ColumnPicker
+        columns={mockColumns}
```

Comment:
need the correct types .... I don't think you're using Column[] really as a type as this doesn't pertain to AgGrid Columns ...? 

The only place where this is used is `DataTable/ColumnPicker` and that's independent from AgGrid itself